packages/devtools_app/lib/src/debugger/flutter/breakpoints.dart

```diff
@@ -114,14 +114,6 @@ class _BreakpointPickerState extends State<BreakpointPicker> {
 
   Future<void> _onBreakpointSelected(BreakpointAndSourcePosition bp) async {
```

Comment:
this method no longer needs to be async.

Reply:
@pq do we have any lints that track when a method body is async but nothing async is actually happening?

Reply:
removed async

Reply:
@jacob314: we don't but it's something to consider.

Tracking issue opened in https://github.com/dart-lang/linter/issues/2087.